soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -4377,7 +4371,7 @@ void KaleidoScope_Update(PlayState* play)
                 VREG(88) = 66;
                 WREG(2) = 0;
                 pauseCtx->alpha = 255;
-                if (!IS_BOSS_RUSH) {
+                if (GameInteractor_Should(VB_BE_ABLE_TO_SAVE, true, NULL)) {
                     pauseCtx->state = 0xE;
                 } else {
                     pauseCtx->state = 0xF;
```

Comment:
When converting things to VB, we want to do our best to remove any additional branches we have added from the source, and instead introduce this behavior when overriding the VB. So in this case (at least if I'm understanding correctly) we would remove this else branch, and in our `hook_handlers` when setting `VB_BE_ABLE_TO_SAVE` to false, we would also set `pauseCtx->state = 0xF;`

Reply:
There can be exceptions to this rule, mostly when dealing with GFX related VB overrides, but in general we want as little of "our code" in the source as possible

Reply:
Yeah, just missed it here

Reply:
Fixed.